File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/model/GLUtils.java
package br.com.insanitech.spritekit.opengl.model;

/**
 * Created by anderson on 7/4/15.
 */
public class GLUtils {
    public static float degree2Rad(float degree) {
        return degree * (float)Math.PI / 180.0f;
    }

    public static float rad2Degree(float rad) {
        return rad * 180.0f / (float)Math.PI;
    }
}

File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/model/GLUtils.kt
package br.com.insanitech.spritekit.opengl.model

/**
 * Created by anderson on 7/4/15.
 */
internal object GLUtils {
    fun degree2Rad(degree: Float): Float = degree * Math.PI.toFloat() / 180.0f
    fun rad2Degree(rad: Float): Float = rad * 180.0f / Math.PI.toFloat()
}
